frame/instrumental-strategy-pablo/src/mock/helpers.rs

```diff
@@ -65,3 +66,8 @@ fn create_pool(
 	assert_ok!(<Pablo as Amm>::add_liquidity(&BOB, pool_id, amounts[0], amounts[1], 0_u128, true));
 	pool_id
 }
+
+pub fn set_admin_account_with_full_access() -> Result<(), ()> {
```

Comment:
I would generalize this so you provide as input (1) an `AccountId` and (2) an `AccessRights` level